lib/ui/window/platform_configuration.h

```diff
@@ -207,6 +207,17 @@ class PlatformConfigurationClient {
   ///
   virtual void RequestDartDeferredLibrary(intptr_t loading_unit_id) = 0;
 
+  //--------------------------------------------------------------------------
+  /// @brief      Invoked when a listener is registered on a platform channel.
+  ///
+  /// @param[in]  name             The name of the platform channel to which a
+  ///                              listener has been registered or cleared.
+  ///
+  /// @param[in]  listening        Whether the listener has been set (true) or
+  ///                              cleared (false).
+  ///
+  virtual void SendChannelUpdate(std::string name, bool listening) = 0;
```

Comment:
This signature doesn't match the implementation. `std::string` vs `const std::string& name`

Reply:
What non-matching implementation to this signature are you looking at? It is implemented in `runtime_controller.cc:405`, where the signatures match.